services/kvstore/src/main/java/tech/pegasys/pantheon/services/kvstore/RocksDbKeyValueStorage.java

```diff
@@ -35,6 +46,144 @@
+    TickerType.NUMBER_DB_PREV, // COUNT: 0
+    TickerType.NUMBER_DB_SEEK_FOUND, // COUNT: 0
+    TickerType.NUMBER_DB_NEXT_FOUND, // COUNT: 0
+    TickerType.NUMBER_DB_PREV_FOUND, // COUNT: 0
+    TickerType.ITER_BYTES_READ, // COUNT: 0
+    TickerType.NO_FILE_CLOSES, // COUNT: 0
+    TickerType.NO_FILE_OPENS, // COUNT: 6
+    TickerType.NO_FILE_ERRORS, // COUNT: 0
+    // TickerType.STALL_L0_SLOWDOWN_MICROS, // COUNT: 0
+    // TickerType.STALL_MEMTABLE_COMPACTION_MICROS, // COUNT: 0
+    // TickerType.STALL_L0_NUM_FILES_MICROS, // COUNT: 0
+    TickerType.STALL_MICROS, // COUNT: 0
+    TickerType.DB_MUTEX_WAIT_MICROS, // COUNT: 0
+    TickerType.RATE_LIMIT_DELAY_MILLIS, // COUNT: 0
+    TickerType.NO_ITERATORS, // COUNT: 0
+    TickerType.NUMBER_MULTIGET_BYTES_READ, // COUNT: 0
+    TickerType.NUMBER_MULTIGET_KEYS_READ, // COUNT: 0
+    TickerType.NUMBER_MULTIGET_CALLS, // COUNT: 0
+    TickerType.NUMBER_FILTERED_DELETES, // COUNT: 0
+    TickerType.NUMBER_MERGE_FAILURES, // COUNT: 0
+    TickerType.BLOOM_FILTER_PREFIX_CHECKED, // COUNT: 0
+    TickerType.BLOOM_FILTER_PREFIX_USEFUL, // COUNT: 0
+    TickerType.NUMBER_OF_RESEEKS_IN_ITERATION, // COUNT: 0
+    TickerType.GET_UPDATES_SINCE_CALLS, // COUNT: 0
+    TickerType.BLOCK_CACHE_COMPRESSED_MISS, // COUNT: 0
+    TickerType.BLOCK_CACHE_COMPRESSED_HIT, // COUNT: 0
+    TickerType.BLOCK_CACHE_COMPRESSED_ADD, // COUNT: 0
+    TickerType.BLOCK_CACHE_COMPRESSED_ADD_FAILURES, // COUNT: 0
+    TickerType.WAL_FILE_SYNCED, // COUNT: 0
+    TickerType.WAL_FILE_BYTES, // COUNT: 246835941
+    TickerType.WRITE_DONE_BY_SELF, // COUNT: 589346
+    TickerType.WRITE_DONE_BY_OTHER, // COUNT: 0
+    TickerType.WRITE_TIMEDOUT, // COUNT: 0
+    TickerType.WRITE_WITH_WAL, // COUNT: 1178692
+    TickerType.COMPACT_READ_BYTES, // COUNT: 112905202
+    TickerType.COMPACT_WRITE_BYTES, // COUNT: 111119756
+    TickerType.FLUSH_WRITE_BYTES, // COUNT: 116155681
+    TickerType.NUMBER_DIRECT_LOAD_TABLE_PROPERTIES, // COUNT: 0
+    TickerType.NUMBER_SUPERVERSION_ACQUIRES, // COUNT: 105
+    TickerType.NUMBER_SUPERVERSION_RELEASES, // COUNT: 0
+    TickerType.NUMBER_SUPERVERSION_CLEANUPS, // COUNT: 0
+    TickerType.NUMBER_BLOCK_COMPRESSED, // COUNT: 86007
+    TickerType.NUMBER_BLOCK_DECOMPRESSED, // COUNT: 281616
+    TickerType.NUMBER_BLOCK_NOT_COMPRESSED, // COUNT: 0
+    TickerType.MERGE_OPERATION_TOTAL_TIME, // COUNT: 0
+    TickerType.FILTER_OPERATION_TOTAL_TIME, // COUNT: 0
+    TickerType.ROW_CACHE_HIT, // COUNT: 0
+    TickerType.ROW_CACHE_MISS, // COUNT: 0
+    TickerType.READ_AMP_ESTIMATE_USEFUL_BYTES, // COUNT: 0
+    TickerType.READ_AMP_TOTAL_READ_BYTES, // COUNT: 0
+    TickerType.NUMBER_RATE_LIMITER_DRAINS, // COUNT: 0
+    TickerType.NUMBER_ITER_SKIP, // COUNT: 0
+    TickerType.NUMBER_MULTIGET_KEYS_FOUND, // COUNT: 0
+  };
+
+  // Histograms - treated as prometheus summaries
+  static final HistogramType[] HISTOGRAMS = {
+    HistogramType.DB_GET, // COUNT : 4716170 SUM : 29514961
```

Comment:
What are all of these comments for?

Reply:
Previous measurements.  I'm was trying to see if there were stats we should get rid of.  But the performance latency impact of keeping all of them is almost unmeasurable.

Reply:
Deleted